TeamCharles/workforce-management
# 51-department-detail-view :: Department detail view

## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.
Adds the detail view for a department.

## Resolves Issue Number
#51 

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout 51-department-detail-view
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1.  dotnet run
2.  navigate to single department view
3.  see list of employees in that department

## Impacted Files in Application
List general components of the application that this PR will affect:

* Views/Department/Detail.cshtml
* ViewModels/Department/DepartmentDetail.cs
* Controllers/DepartmentController.cs
